src/config/constants.ts

```diff
@@ -185,13 +185,19 @@ export const HOST_VALIDATION_FAILURE_HINTS: { [key in HostValidationId]: string
     'Please manually fix hosts NTP configuration or provide Additional NTP source in Advanced networking configuration section.',
 };
 
-export const CLUSTER_DEFAULT_NETWORK_SETTINGS = {
+export const CLUSTER_DEFAULT_NETWORK_SETTINGS_IPV4 = {
   clusterNetworkCidr: '10.128.0.0/14',
   clusterNetworkHostPrefix: 23,
   serviceNetworkCidr: '172.30.0.0/16',
   additionalNtpSource: '0.rhel.pool.ntp.org',
```

Comment:
maybe we should put the additionalNtpSource default into a separate constant? Something like
```
export const CLUSTER_DEFAULT_SETTINGS: = {
  additionalNtpSource: '0.rhel.pool.ntp.org',
}

...
export const getClusterDefaultSettings = (clusterNetworkCidr: Cluster['clusterNetworkCidr']) => ({
  ...CLUSTER_DEFAULT_SETTINGS,
  ...getDefaultNetworkSettings(cluster.clusterNetworkCidr as string);
})

Reply:
no problem